src/dr_wandb/run_metadata.py

```
@@ -249,23 +249,45 @@ def load_canonical_run_metadata(
-    canonical_runs = sorted(canonical_runs, key=lambda record: record.run_id)
-    canonical_runs = sorted(
-        canonical_runs,
-        key=lambda record: record.createdAt or "",
+    raw_records = sorted(latest_by_run_id.values(), key=lambda record: record.run_id)
+    raw_records = sorted(
+        raw_records,
+        key=lambda record: _coerce_str(record.raw_run.get("createdAt")) or "",
         reverse=True,
     )
-    return canonical_runs
+    return raw_records
```

Ruff (0.15.6)

[warning] Unnecessary assignment to `raw_records` before `return` statement

Remove unnecessary assignment

(RET504)